src/tike/view.py

```diff
@@ -58,12 +58,108 @@
 import matplotlib.transforms as transforms
 from matplotlib import collections
 import matplotlib.pyplot as plt
+import matplotlib.colors as mplcolors
+import cv2 as cv
 import numpy as np
 import tike.linalg
 
 logger = logging.getLogger(__name__)
 
 
+def complexHSV_to_RGB(img0):
+    """Convert a complex valued array to RGB representation.
+    
+    Takes a complex valued 2D array, represents the phase as hue,
```

Comment:
```suggestion
    Takes a complex valued array, represents the phase as hue,
```
Array does not need to be 2D.